sample.html

```diff
@@ -74,10 +74,22 @@
             window.pp.mb.publish(OO.EVENTS.WIKIA.SHOW_AD_FULLSCREEN_TOGGLE, false);
           }
         }, false, this);
+
+        // that's a hack for autoplay on mobile for VPAID ads
```

Comment:
and by not perfectly? what does that mean?
can we have a small list what is not working?
will we remember in 3 months what that means?